### DISCUSSION TITLE
Configure aio setup password via docker secrets

### DISCUSSION THREAD

**Discussion by dantefromhell** [nextcloud/all-in-one#804] [Ideas] [answered]
I am deploying the nextcloud aio container within a longer ansible playbook.

Ansible has a way to manage secrets + a module to deploy them into docker secrets (considering docker env variables to be insecure for secrets).

It would be great if there's a way to tell the mastercontainer to source the password from docker secrets to enable devops process w/ managed secrets.

**Comment by szaimen** [comment]
I will not work on this but PRs are welcome!

**Comment by szaimen** [comment, marked as answer]
Lets track this in https://github.com/nextcloud/all-in-one/discussions/920
